tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -0,0 +1,206 @@
+assert = chai.assert
+
+th = new Annotator.Plugin.TextHighlights()
+th.pluginInit()
+
+class FakeAnchor extends Annotator.Anchor
+
+  _getSegment: -> "html segment for " + @id
+
+  constructor: (manager, annotation, target) ->
+    super manager, annotation, target, 42, 42,
+      "fake quote for" + target.id
+
+    @id = "fake anchor for " + target.id
+
+describe 'Annotator.Plugin.EnhancedAnchoring', ->
+  sandbox = null
+  annotator = null
+  data = null
+  ann1 = null
+  ann2 = null
+  ann3 = null
+  target11 = null
+  target12 = null
+  target2 = null
+  target3 = null
+  anchor11 = null
+  anchor12 = null
+  anchor2 = null
+  anchor3 = null
+
+  setupManager = ->
+    annotator =
+      publish: (event) ->
+
+    am = new Annotator.Plugin.EnhancedAnchoring()
```

Comment:
On 2015-02-05 17:31, Aron Carroll wrote:
> > +    am = new Annotator.Plugin.EnhancedAnchoring()
>
> I would expect this method to describe the object it's returning, eg.
> |createAnchoringPlugin|. It's currently unclear what |setupManager|
> and |am| are.
>

I renamed setupManager() to createAnchoringManager().
(Hopefully, that will make it obvious that "am" stands for "anchoring
manager".)